docs/cloud/managing-airbyte-cloud.md

```diff
@@ -276,7 +278,70 @@ To refresh the source schema:
 
 2. If there are changes to the schema, the **Refreshed source schema** dialog displays them.
 
+## Review schema changes
+Airbyte detects changes in your source schema and allows you to review the changes. 
+
+### Non-breaking changes
+
+To review non-breaking schema changes:
+1. On the [Airbyte Cloud](http://cloud.airbyte.io/) dashboard, click **Connections** and select the connection with non-breaking changes (indicated by a **yellow exclamation mark** icon).
+
+2. Click **Review changes**.
+
+3. The **Refreshed source schema** dialog displays the changes. 
+
+4. Click **OK** to close the dialog.
+
+5. Scroll to the bottom of the page and click **Save changes**.
+
+:::note 
+    
+ By default, Airbyte ignores non-breaking changes and continues syncing. You can configure how Airbyte handles syncs when it detects non-breaking changes by [editing the stream configuration](#edit-stream-configuration).
+    
+:::
+
+### Breaking changes
+
+To review breaking schema changes:
```

Comment:
Change "To review breaking schema changes:" > "To review and fix breaking schema changes:"